<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Particula</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="8" column="0" colspan="3">
        <widget class="QPushButton" name="agregarFinal_pushButton">
         <property name="text">
          <string>Agregar Final</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Destino Y</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QSpinBox" name="red_spinBox">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Green</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="destinoX_spinBox">
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Destino X</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Velocidad</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QSpinBox" name="destinoY_spinBox">
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="3">
        <widget class="QPushButton" name="mostrar_pushButton">
         <property name="text">
          <string>Mostrar</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="3">
        <widget class="QPushButton" name="agregarInicio_pushButton">
         <property name="text">
          <string>Agregar Inicio</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Red</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QSpinBox" name="velocidad_spinBox">
         <property name="maximum">
          <number>300</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Blue</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QSpinBox" name="origenX_spinBox"/>
       </item>
       <item row="7" column="2">
        <widget class="QSpinBox" name="blue_spinBox">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QSpinBox" name="green_spinBox">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="origenY_spinBox"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Origen X</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Origen Y</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPlainTextEdit" name="salida"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>504</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
